catboost/pytest/test.py

```diff
@@ -8516,3 +8516,32 @@ def test_model_shrink_incorrect(config):
     ]
     with pytest.raises(yatest.common.ExecutionError):
         yatest.common.execute(cmd)
+
+
+def test_total_f1_params():
```

Comment:
It will be better to make `average` as parameter of test and return result of `do_test_eval_metrics` with canonization.